settings/UL/settings_embeddingselection_2016postVFPUL.yaml

```diff
@@ -1,9 +1,22 @@
 ID_pt_bins_inc_eta:
-  # Settings for histogram creation
   name: "ID_pt_bins_inc_eta"
   var: "m_vis(100,50,150)"
-  tag: "(trg_IsoMu22_tag || trg_IsoTkMu22_tag || trg_IsoMu22_eta2p1_tag || trg_IsoTkMu22_eta2p1_tag) && fsr_photon_veto_tag < 0.5 && fsr_photon_veto_probe < 0.5 && pt_tag > 23 && iso_tag < 0.15 && dz_tag < 0.2 && dxy_tag < 0.045 && id_medium_tag"
-  probe: "id_medium_probe"
+  tag:  (
```

Comment:
should we drop these inc measurements, that are never used?

Reply:
the `id_medium` is anyhow never used here

Reply:
As you say, I suggest to do this mainly to make the configs shorter and easier to maintain

Reply:
Should I then also propagate this change to settings_muon_{era}.yaml, since we also do the inclusive sf measurements there?